client/app/bundles/course/assessment/submission/containers/ProgrammingImportEditor.jsx

```diff
@@ -21,142 +20,193 @@ const styles = {
   },
 };
 
-class VisibleProgrammingImportEditor extends Component {
-  static renderSelectProgrammingFileEditor(props) {
-    const { fields, readOnly, language, displayFileIndex } = props;
+const renderSelectProgrammingFileEditor = (props) => {
```

Comment:
Same comment as before with respect to naming and usage, since you're using hooks inside.